includes/definitions/gw-eydfa.yaml

```diff
@@ -0,0 +1,79 @@
+os: gw-eydfa
```

Comment:
Probably can just be `eydfa`

Reply:
eydfa is the kind of hardware, too generic for being an OS I think.

Reply:
No name for the firmware they run or the product range?

Reply:
Stick with gw-eydfa then